bin/resources/GameIndex.yaml

```diff
@@ -17165,7 +17207,7 @@ SLES-53866:
   speedHacks:
     MTVUSpeedHack: 0 # Fixes bad graphics due to bad T-Bit handling.
 SLES-53867:
-  name: "Ski Alpin 2006"
+  name: "Alpine Skiing 2006 featuring Bode Miller"
```

Comment:
```suggestion
  name: "Bode Miller Alpine Skiing"
```

Reply:
No, "Bode Miller Alpine Skiing" is the NTSC-U title.

There is 2 cover titles for SLES-53867:
- "Alpine Skiing 2006 featuring Bode Miller" (for the PAL-E cover)
- "Ski Alpin 2006 mit Bode Miller" (for the PAL-G cover)

Reply:
ah  my bad , my mistake then  :+1: